src/query/api/v1/options/handler.go

```diff
@@ -199,6 +226,8 @@ type handlerOptions struct {
 	placementServiceNames []string
 	serviceOptionDefaults []handleroptions.ServiceOptionsDefault
 	nowFn                 clock.NowFn
+	queryRouter           QueryRouter
+	instantQueryRouter    QueryRouter
```

Comment:
Unfortunately github would let me add comments on unchanged code so I could put it in the relevant place, but we should set the defaults for these inside of `NewHandlerOptions`, e.g. like [we do here](https://github.com/m3db/m3/blob/master/src/query/models/options.go#L31)

Also could be a bit of a problem in the future that these Options don't have a `Validate() error` method, but that's entirely my fault :p